Paraswap-Agents/AdminOperations/src/findings.ts

```diff
@@ -0,0 +1,81 @@
+import { LogDescription, TransactionDescription } from "ethers/lib/utils";
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+
+export const createEventFinding = (log: LogDescription) => {
+  let name = log.name == "RouterInitialized" ? "Router" : "Adapter";
+  return Finding.fromObject({
+    name: `Admin operation detected: ${name} has been initialized`,
+    description: `${log.name} event was emitted in AugustusSwapper contract`,
+    alertId: "PARASWAP-1-1",
+    severity: FindingSeverity.Info,
+    type: FindingType.Info,
+    protocol: "Paraswap",
+    metadata: {
+      address: log.args.router ? log.args.router.toLowerCase() : log.args.adapter.toLowerCase(),
+    },
+  });
+};
+
+export const createFunctionFinding = (call: TransactionDescription) => {
```

Comment:
I think it would be good to have an exclusive Finding for when it is an ETH transfer. With this current `transferTokens` Finding, we wouldn't know it is an ETH transfer except for the `token` in the `metadata` would be `0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE`.

Perhaps an ETH transfer Finding could be:
```
return Finding.fromObject({
        name: `Admin operation detected: ETH was transferred`,
        description: `${call.name} function was called in AugustusSwapper contract`,
        alertId: "PARASWAP-1-3", // adjust the rest
        severity: FindingSeverity.Info,
        type: FindingType.Info,
        protocol: "Paraswap",
        metadata: {
          token: "ETH",
          destination: call.args.destination.toLowerCase(),
          amount: call.args.amount.toString(),
        },
      });
```
Then adding the appropriate unit test.